src/Api/Endpoints/MetaDataService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Passwordless.Api.Authorization;
+using Passwordless.Common.Models.MDS;
+using Passwordless.Service.Features;
+using Passwordless.Service.Helpers;
+using Passwordless.Service.MDS;
+using static Microsoft.AspNetCore.Http.Results;
+
+namespace Passwordless.Api.Endpoints;
+
+public static class MetaDataServiceEndpoints
+{
+    public static void MapMetaDataServiceEndpoints(this WebApplication app)
+    {
+        var parent = app
+            .MapGroup("/mds")
+            .RequireSecretKey()
+            .RequireCors("default");
```

Comment:
praise: Nice use of groups
nitpick: renam `parent` => `mds`.

thought: we should do a team API review for this as well. If these are assumed to be internal, we might need to stark indicate that in code, for us to keep track and in the future to avoid listing them in OpenAPI specs.